docs/getting-started/getting-started-sql-snowflake.md

```diff
@@ -272,50 +284,121 @@ Clicking on any of the Gems shows these options.
+
+#### 4.5.1 Test
+
+You'll want to know how the data looks after each step, not just for the final table or view.
 
 ![Interactively run](img/4-9-interactively-run.png)
 
 Prophecy makes interactively testing the models incredibly easy! Simply click on the **(1) Play** button on any of the Gems and the model with all of it’s upstream dependencies will be executed. Once the model runs, the **(2) Result** icon appears. Click the Result icon to view a **(3) Sample** set of records.
 
+##### 4.5.2 FixIt
+
+If your model has some errors, perhaps introduced upstream without your knowledge, Prophecy's Copilot will suggest fixes automatically.
+
+<div class="wistia_responsive_padding" style={{padding:'56.25% 0 0 0', position:'relative'}}>
+<div class="wistia_responsive_wrapper" style={{height:'100%',left:0,position:'absolute',top:0,width:'100%'}}>
+<iframe src="https://fast.wistia.net/embed/iframe/17q9stsz6n?videoFoam=true" title="Drag and Drop Model Graph Video" allow="autoplay; fullscreen" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" msallowfullscreen width="100%" height="100%"></iframe>
+</div></div>
+<script src="https://fast.wistia.net/assets/external/E-v1.js" async></script>
+
+The Fix with AI option appears at any point where you see an error message.
+
+## 5. Code view
+
+The visual developers will appreciate the drag-n-drop canvas, but sometimes it's also nice to view the code. Already Prophecy is creating highly performant code behind the scenes. Just click the **Code View** to reveal the SQL queries we've generated using our visual design editor. Each Gem is represented by a CTE or subquery. For example, the Join Gem `NATIONS_CUSTOMERS` is highlighted in both visual and code views.
+
+![ToggleToCode](./img/Snow4.6_toggleVisualCode.png)
+
+You may wish to edit the code view - give it a try! Add a SQL statement in the code view and notice the visual editor displays the updated code. For example, we've added a limit statement in the code view, and a new limit Gem appears in the visual view.
+
+<div class="wistia_responsive_padding" style={{padding:'56.25% 0 0 0', position:'relative'}}>
+<div class="wistia_responsive_wrapper" style={{height:'100%',left:0,position:'absolute',top:0,width:'100%'}}>
+<iframe src="https://fast.wistia.net/embed/iframe/gpb6yu0bxv?videoFoam=true" title="Edit Code Video" allow="autoplay; fullscreen" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" msallowfullscreen width="100%" height="100%"></iframe>
+</div></div>
+<script src="https://fast.wistia.net/assets/external/E-v1.js" async></script>
+
+Coming soon, you'll see code suggestions in the code editor. Copilot helps write SQL queries in the code editor and in the visual editor.
+
+![CodeSuggestions](./img/SnowCodeSuggestions.png)
+
+## 6. Team considerations
```

Comment:
I'd focus on scheduling more than documentation. I thought we were supposed to discuss Airflow, but that seems to be missing?

Reply:
Auto-docs can again just be a link to co-pilot documentation.